### PR TITLE
Fixed command /usr/bin/thermalctld to /usr/local/bin/thermalctld in platform test

### PR DESCRIPTION
**Pull request by andriyz-nv**
### Description of PR
Fixed command /usr/bin/thermalctld to /usr/local/bin/thermalctld
The file path was changed in bellow pull request, but tests in
sonic-mgmt were not updated, after that PR was merged
https://github.com/Azure/sonic-buildimage/pull/6176

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x ] Test case(new/improvement)

#### How did you verify/test it?
Executed test, test passed without
_fan_log_supported = {'stderr_lines': [u'grep: /usr/bin/thermalctld: No such file or directory'], u...: [], u'start': u'2021-03-17 21:37:36.289661', u'msg': u'non-zero return code'}

### CODE REVIEW

File: tests/platform_tests/test_platform_info.py [hunk 1]
```diff
@@ -439,7 +439,7 @@ def test_thermal_control_fan_status(duthosts, rand_one_dut_hostname, mocker_fact
         single_fan_mocker = mocker_factory(duthost, 'SingleFanMocker')
         time.sleep(THERMAL_CONTROL_TEST_WAIT_TIME)
 
-        _fan_log_supported = duthost.command('docker exec pmon grep -E "{}" /usr/bin/thermalctld'\
+        _fan_log_supported = duthost.command('docker exec pmon grep -E "{}" /usr/local/bin/thermalctld'\
```
Comment (jleveque) [inline tests/platform_tests/test_platform_info.py:442]: Thanks for catching and fixing! However, these tests are expected to run on older versions of SONiC, before the path was changed. Thus I suggest something like the following:

```python
        if "201811" in dut.os_version or "201911" in dut.os_version:
            THERMALCTLD_PATH = '/usr/bin/thermalctld'
        else:
            THERMALCTLD_PATH = '/usr/local/bin/thermalctld'

        _fan_log_supported = duthost.command('docker exec pmon grep -E "{}" {}'
                .format(LOG_EXPECT_INSUFFICIENT_FAN_NUM_RE, THERMALCTLD_PATH), module_ignore_errors=True)
```

**Review by jleveque — CHANGES_REQUESTED** [review]

**Comment by wangxin** [comment]
@andriyz-nv, Can you address the review comments?

File: tests/platform_tests/test_platform_info.py (hunk 1 above)
Comment (andriyz-nv) [reply tests/platform_tests/test_platform_info.py]: Hi @jleveque 
Can you please review new

**Review by andriyz-nv — COMMENTED** [review]
@jleveque  I fixed according to your comments

**Review by jleveque — APPROVED** [review]

**Comment by andriyz-nv** [comment]
@wangxin can you please review?

**Review by wangxin — APPROVED** [review]

**Pull request merged**